toolchain/sem_ir/ids.h

```diff
@@ -601,13 +601,18 @@ struct ElementIndex : public IndexBase, public Printable<ElementIndex> {
 };
 
 // The ID of an ImportIRInst.
-struct ImportIRInstId : public IdBase, public Printable<InstId> {
+struct ImportIRInstId : public IdBase, public Printable<ImportIRInstId> {
   using ValueType = ImportIRInst;
 
   // An explicitly invalid ID.
   static const ImportIRInstId Invalid;
 
   using IdBase::IdBase;
+
+  auto Print(llvm::raw_ostream& out) const -> void {
+    out << "import_ir_inst";
```

Comment:
We seem to have no test coverage for this; can you add some?

Reply:
Done. Also started printing the import_ir_inst list, and adjusted formatting for that and ImportIR.